### PR TITLE
[WEB-3293] Handle null response at YML validation

### PR DESCRIPTION
**Pull request by slapec93**

### CODE REVIEW

File: source/javascripts/services/_appService.js.erb [hunk 1]
```diff
@@ -334,6 +334,9 @@ angular.module("BitriseWorkflowEditor").service("appService", function($q, reque
 			appService.savedAppConfig = undefined;
 			appService.savedAppConfigYML = appService.appConfigYML;
 
+			if (validationData.warnings === null) {
```
Comment (tadam313) [inline source/javascripts/services/_appService.js.erb]: can we change it to `!validationData.warnings`

File: apiserver/utility/utility.go [hunk 2]
```diff
@@ -44,12 +45,15 @@ func ValidateBitriseConfigAndSecret(bitriseConfig, secretsConfig string) (*Warni
 		return nil, fmt.Errorf("Validation failed: %s", strings.Join(errorStrs, " | "))
 	}
 
-	warningItems := &WarningItems{Config: []string{}, Secrets: []string{}}
+	warningItems := WarningItems{}
 	if len(bitriseWarns) > 0 {
 		warningItems.Config = bitriseWarns
 	}
```
Comment (tadam313) [inline apiserver/utility/utility.go]: can't we do? `if len(bitriseWarns) > 0 {} else { return nil, nil }` instead of using `reflect`?

**Comment by tadam313** [comment]
@slapec93 also go testing failed

**Comment by slapec93** [comment]
Reopening in other PR

**Pull request closed without merging**